lib/sprinkler_web/live/dashboard_live.ex

```diff
@@ -0,0 +1,19 @@
+defmodule SprinklerWeb.DashboardLive do
+  use SprinklerWeb, :live_view
+
+ @telemetry_topic "telemetry"
+
+  @impl true
+  def mount(_params, _session, socket) do
+    Phoenix.PubSub.subscribe(Sprinkler.PubSub, @telemetry_topic)
+    {:ok, assign(socket, :devices,[device_1: %{id: 1, tmps: [2]}])}
```

Comment:
I'm nos sure about this stricture. I wanted one that could be easily accessed for reading and also updating according to the device id 😕